PROPOSALCOPY.MD

```diff
@@ -0,0 +1,42 @@
+## Proposal to use a bot to deposit keys and ether to contract.
+
+As you know, to keep our stacking protocol working we should deposit keys and ether to deposit contract.
+For now, we don't have any automatic solution.
+
+We propose:
+1. Run and fund bot that will deposit buffered ether to contract.
+2. Make refund to the development team that deposited buffered ether to contract before.
+
+## Problem
+
+Ether is placed in buffer, and it must be deposited by calling smart contract. Until now, it was done manually.
+Manual action requires attention, and we can save money if we deposit while the cost of gas is low.
+
+## Strategy
+
+The main idea is to control gas fee and create transaction when the gas fee is quite low.
```

Comment:
It's better not to use relative terms without numbers.

Reply:
Another thing, if that's a section to outline suggested solution, we may write out what the solution is — not the gas strategy per se, but the bot utilizing it + the infra required to get the relevant gas price data